local-interchaintest/src/utils.rs

```diff
@@ -0,0 +1,161 @@
+use orbital_core::{
+    msg::{ExecuteMsg, QueryMsg},
+    orbital_domain::UncheckedOrbitalDomainConfig,
+    state::UserConfig,
+};
+
+use crate::ACC0_KEY;
+
+pub fn query_user_clearing_acc_addr_on_domain(
+    test_ctx: &TestContext,
+    orbital_core: String,
+    user_addr: &str,
+    domain: String,
+) -> Result<Option<String>, LocalError> {
+    let clearing_acc_response = contract_query(
+        test_ctx
+            .get_request_builder()
+            .get_request_builder(NEUTRON_CHAIN_NAME),
+        &orbital_core,
+        &serde_json::to_string(&QueryMsg::ClearingAccountAddress {
+            addr: user_addr.to_string(),
+            domain: domain.to_string(),
+        })
+        .map_err(|e| LocalError::Custom { msg: e.to_string() })?,
+    )["data"]
+        .clone();
+    let user_clearing_acc: Option<String> = serde_json::from_value(clearing_acc_response)
+        .map_err(|e| LocalError::Custom { msg: e.to_string() })?;
+
+    info!(
+        "user {user_addr} clearing account on {domain}: {:?}",
+        user_clearing_acc
+    );
+
+    Ok(user_clearing_acc)
+}
+
+pub fn query_user_config(
+    test_ctx: &TestContext,
+    orbital_core: String,
+    user_addr: &str,
+) -> Result<UserConfig, LocalError> {
+    let query_response = contract_query(
+        test_ctx
+            .get_request_builder()
+            .get_request_builder(NEUTRON_CHAIN_NAME),
+        &orbital_core,
+        &serde_json::to_string(&QueryMsg::UserConfig {
+            addr: user_addr.to_string(),
+        })
+        .map_err(|e| LocalError::Custom { msg: e.to_string() })?,
+    )["data"]
+        .clone();
+
+    let user_config: UserConfig = serde_json::from_value(query_response).unwrap();
+
+    info!("user {user_addr} config: {:?}", user_config);
+
+    Ok(user_config)
+}
+
+pub fn query_registered_users(
+    test_ctx: &TestContext,
+    orbital_core: String,
+) -> Result<(), LocalError> {
+    let registered_users_query_response = contract_query(
+        test_ctx
+            .get_request_builder()
+            .get_request_builder(NEUTRON_CHAIN_NAME),
+        &orbital_core,
+        &serde_json::to_string(&QueryMsg::UserAddresses {})
+            .map_err(|e| LocalError::Custom { msg: e.to_string() })?,
+    )["data"]
+        .clone();
+
+    info!(
+        "registered users query response: {:?}",
+        registered_users_query_response
+    );
+    Ok(())
+}
+
+pub fn user_register_orbital_core(
+    test_ctx: &TestContext,
+    user_key: &str,
+    orbital_core: String,
+) -> Result<TransactionResponse, LocalError> {
+    info!("registering user {user_key} to orbital-core...");
+    contract_execute(
+        test_ctx
+            .get_request_builder()
+            .get_request_builder(NEUTRON_CHAIN_NAME),
+        &orbital_core,
+        user_key,
+        &serde_json::to_string(&ExecuteMsg::RegisterUser {})
+            .map_err(|e| LocalError::Custom { msg: e.to_string() })?,
+        "--amount 1000000untrn --gas 5000000",
```

Comment:
nit: reuse GAS_FLAGS? or maybe they are not even needed

Reply:
removed this entirely, wasn't needed